src/utils/ui-utils.ts

```diff
@@ -0,0 +1,175 @@
+import ResizeSensor from 'css-element-queries/src/ResizeSensor';
+
+/**
+ * Some of the tables can be very long and the horizontal scroll only sits at the very bottom by default
```

Comment:
In the comment explain that this is similar to the chaise function and mention why you had to customize it instead of just using the chaise function.. Later I can change change the chaise function to handle this scenario as well.

Also please add inline comments in the code with `/** different from chaise **/` to point the actual code lines that are different.

You should also remove the `console.log`s from these functions.